### PR TITLE
language pack modified 19/09/2022

### PR DESCRIPTION
**Pull request by ashahrior**
@NavidMahbub

### CODE REVIEW

File: lang/bn/forum.php [hunk 1]
```diff
@@ -332,7 +332,7 @@
 $string['shortpost'] = 'ছোট পোস্ট';
 $string['showsubscribers'] = 'প্রদর্শন/সম্পাদন বর্তমান সাবস্ক্রাইবার';
 $string['singleforum'] = 'একটি সহজ আলোচনা';
-$string['smallmessage'] = '{$a->forumname} এ পোস্টকৃত {$a->user}';
+$string['smallmessage'] = '{$a->forumname} এ পোস্টকৃত {$a->ব্যবহারকারী}';
```
Comment (ashahrior) [inline lang/bn/forum.php:335]: revert to
```suggestion
$string['smallmessage'] = '{$a->forumname} এ পোস্টকৃত {$a->user}';
```

File: lang/bn/forum.php [hunk 2]
```diff
@@ -233,7 +233,7 @@
 $string['movedmarker'] = '(সরানো)';
 $string['movethisdiscussionto'] = 'আলোচনাটি সরানো...';
 $string['mustprovidediscussionorpost'] = 'এক্সপোর্ট করতে আপনাকে হয় আলোচনার আইডি অথবা পোস্ট আইডি দিতে হবে';
-//$string['namenews'] = 'সংবাদের ফোরাম';
+$string['namenews'] = 'সংবাদের ফোরাম';
```
Comment (ashahrior) [inline lang/bn/forum.php:236]: check why this was commented.

File: lang/bn/moodle.php [hunk 3]
```diff
@@ -1068,7 +1068,7 @@
 $string['newpicture'] = 'নতুন ছবি';
 $string['newpicture_help'] = 'নতুন ছবি যোগ করতে, ব্রাউজ করুন এবং চিত্র নির্বাচন করুন (JPG অথবা PNG বিন্যাসে) তারপর "হালনাগাদ প্রোফাইলে" ক্লিক করুন। চিত্রটি বর্গক্ষেত্ররুপে ছাঁটতে হবে এবং পুনরাকৃতি হবে ১০০x১০০ পিক্সেলে।';
 $string['newsitem'] = 'সংবাদ আইটেম';
-//$string['newsitems'] = 'সংবাদ আইটেম';
+$string['newsitems'] = 'সংবাদ আইটেম';
```
Comment (ashahrior) [inline lang/bn/moodle.php:1071]: same as above.

File: lang/bn/moodle.php [hunk 4]
```diff
@@ -1557,7 +1563,6 @@
 $string['statsuserreads'] = 'মতামত';
 $string['statsuserwrites'] = 'পোষ্ট';
 $string['statswrites'] = 'পোষ্ট';
-$string['status'] = 'অবস্থা';
```
Comment (ashahrior) [inline lang/bn/moodle.php:1560]: this was introduced.

File: lang/bn/moodle.php [hunk 5]
```diff
@@ -1770,22 +1775,79 @@
...
-$string['enddate'] = 'কোর্স শেষের তারিখ';
-$string['courseoverviewfiles'] = 'কোর্স ইমেজ';
 $string['appearance'] = 'অ্যাপিয়ারেন্স';
-$string['filesanduploads'] = 'ফাইল এবং আপলোড';
 $string['contentbank'] = 'কন্টেন্ট ব্যাঙ্ক';
-$string['coursecompletion'] = 'কোর্স সমাপ্তি';
-$string['coursereuse'] = 'কোর্স পুনঃব্যবহার';
-$string['showoncoursepage'] = 'কোর্স পৃষ্ঠায় দেখান';
 $string['hidefromstudents'] = 'প্রশিক্ষণার্থীদের থেকে লুকান';
-$string['settings'] = 'সেটিংস';
```
Comment (ashahrior) [inline lang/bn/moodle.php:1788]: also check these.

File: lang/bn/forum.php (hunk 1 above)
Comment (NavidMahbub) [reply lang/bn/forum.php]: Modified

File: lang/bn/forum.php (hunk 2 above)
Comment (NavidMahbub) [reply lang/bn/forum.php]: checked. It was duplicated. commented out again

File: lang/bn/qtype_match.php [hunk 6]
```diff
@@ -31,4 +31,11 @@
...
-$string['correctansweris'] = 'সঠিক উত্তর: {$a}';
\ No newline at end of file
+$string['correctansweris'] = 'সঠিক উত্তর: {$a}';
+$string['pluginname'] = 'ম্যাচিং';
+$string['pluginnamesummary'] = 'কয়েকটি সাব-প্রশ্নের প্রতিটির উত্তর অবশ্যই সম্ভাবনার তালিকা থেকে বেছে নিতে হবে।';
+$string['availablechoices'] = 'উপলভ্য পছন্দসমূহ';
+$string['pluginnameadding'] = 'ম্যাচিং প্রশ্ন যোগ করা';
+$string['pluginnameediting'] = 'ম্যাচিং প্রশ্ন সম্পাদনা করা';
+$string['pluginname_help'] = 'সমস্যার প্রশ্নগুলির জন্য উত্তরদাতাকে নাম বা বিবৃতিগুলির (প্রশ্ন) তালিকার অন্য নাম বা বিবৃতির (উত্তর) তালিকার সাথে সঠিকভাবে মেলাতে হবে';
+$string['blanksforxmorequestions'] = 'আরো {কোন} প্রশ্নের জন্য ফাঁকা';
```
Comment (ashahrior) [inline lang/bn/qtype_match.php:41]: `{কোন}`?

File: lang/bn/quiz.php [hunk 7]
```diff
@@ -706,10 +706,33 @@
 $string['overduehandlingautoabandon'] = 'সময় শেষ হওয়ার আগেই প্রয়াস জমা দিতে হবে, অথবা সেগুলি গণনা করা হবে না';
 $string['backtocourse'] = 'কোর্সে ফিরুন';
 $string['questionbank'] = 'প্রশ্ন ব্যাঙ্ক থেকে';
-$string['add'] = 'যোগ';
```
Comment (ashahrior) [inline lang/bn/quiz.php:709]: this.

File: lang/bn/grading.php [hunk 8]
```diff
@@ -26,4 +26,74 @@
...
+
+ডিফল্টরূপে, শুধুমাত্র ভাগ করা টেমপ্লেট হিসাবে সংরক্ষিত গ্রেডিং ফর্মগুলি অনুসন্ধান ফলাফলে অন্তর্ভুক্ত করা হয়৷ আপনি অনুসন্ধান ফলাফলে আপনার নিজস্ব গ্রেডিং ফর্ম অন্তর্ভুক্ত করতে পারেন। এইভাবে, আপনি আপনার গ্রেডিং ফর্মগুলি ভাগ না করেই পুনরায় ব্যবহার করতে পারেন৷ শুধুমাত্র \'ব্যবহারের জন্য প্রস্তুত\' হিসাবে চিহ্নিত ফর্মগুলি এইভাবে পুনরায় ব্যবহার করা যেতে পারে।';
+$string['searchownforms'] = 'আমার নিজস্ব ফর্ম অন্তর্ভুক্ত করুন';
+$string['statusdraft'] = 'খসড়া';
+$string['statusready'] = 'ব্যবহারের জন্য প্রস্তুত';
+$string['templatedelete'] = 'মুছুন';
+$string['templatedeleteconfirm'] = 'আপনি শেয়ার করা টেমপ্লেট মুছে ফেলতে যাচ্ছেন \'{$a}\'। একটি টেমপ্লেট মুছে ফেলা বিদ্যমান ফর্মগুলিকে প্রভাবিত করে না যা এটি থেকে তৈরি করা হয়েছিল৷';
+$string['templateedit'] = 'সম্পাদনা';
+$string['templatepick'] = 'এই টেমপ্লেটটি ব্যবহার করুন';
+$string['templatepickconfirm'] = 'আপনি কি গ্রেডিং ফর্ম \'{$a->component}-এ নতুন গ্রেডিং ফর্মের টেমপ্লেট হিসেবে \'{$a->formname}\' ব্যবহার করতে চান ({$a ->এরিয়া})\'?';
```
[the remark this answers is not in the document]
Comment (NavidMahbub) [reply lang/bn/grading.php:93]: Modified

File: lang/bn/moodle.php [hunk 9]
```diff
@@ -692,7 +692,7 @@
 $string['frontpagedescriptionhelp'] = 'সাইটের এই বর্ণনা প্রদর্শিত হবে সম্নুখ পৃষ্ঠায়';
 $string['frontpageformat'] = 'সম্নুখ পৃষ্ঠার বিন্যাস';
 $string['frontpageformatloggedin'] = 'লগইনের সময় সম্নুখ পৃষ্ঠার বিন্যাস ';
-//$string['frontpagenews'] = 'নিউজ আইটেম';
+$string['frontpagenews'] = 'নিউজ আইটেম';
```
[the remark this answers is not in the document]
Comment (NavidMahbub) [reply lang/bn/moodle.php:695]: Don't know. But need this now.

File: lang/bn/user.php [hunk 10]
```diff
@@ -32,6 +32,8 @@
 $string['clearfilters'] = 'ফিল্টার ক্লিয়ার করুন';
 $string['applyfilters'] = 'ফিল্টার করুন';
 $string['addcondition'] = 'শর্ত যোগ করুন';
-$string['countparticipantsfound'] = '{$a} জন অংশগ্রহণকারী প্রাপ্ত';
+$string['countparticipantsfound'] = '{$a} অংশগ্রহণকারী পাওয়া গেছে';
```
Comment (ashahrior) [inline lang/bn/user.php:35]:
```suggestion
$string['countparticipantsfound'] = '{$a} জন অংশগ্রহণকারী পাওয়া গেছে';
```

File: lang/bn/moodle.php (hunk 3 above)
Comment (NavidMahbub) [reply lang/bn/moodle.php:1071]: Need both `$string['newsitem']` & `$string['newsitems']`. Label looks same but selector is dif...

File: lang/bn/moodle.php (hunk 4 above)
Comment (NavidMahbub) [reply lang/bn/moodle.php:1560]: `$string['status']` exits in `moodle.php` with another value

File: lang/bn/moodle.php (hunk 5 above)
Comment (NavidMahbub) [reply lang/bn/moodle.php:1788]: Added in bottom

File: lang/bn/qtype_match.php (hunk 6 above)
Comment (NavidMahbub) [reply lang/bn/qtype_match.php:41]: modified

File: lang/bn/quiz.php (hunk 7 above)
Comment (NavidMahbub) [reply lang/bn/quiz.php:709]: added latter

File: lang/bn/user.php (hunk 10 above)
Comment (NavidMahbub) [reply lang/bn/user.php:35]: modified

**Pull request merged**